README.md

```diff
@@ -64,6 +64,14 @@ To run a particular test, say `test_whatweekisit.py`, run:
 poetry run pytest tests\test_whatweekisit.py
 ```
 
+## Code Styling
+
+We use an automated code formatter called [Black](https://black.readthedocs.io/), currently this needs to be run manually to pass the format CI check. To run Black, run from the root of the repo:
+
+```bash
+poetry run black uqcsbot
```

Comment:
does that work? poetry just inherently knows which files to do? massive

Reply:
Well in this case, its running it on the folder uqcsbot so it's smart enough to know what files to run it on otherwise